portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java

```diff
@@ -23,18 +23,7 @@
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.net.URLEncoder;
-import java.util.ArrayList;
```

Comment:
Can you not cleanup imports as this shouldn't be needed for this change and makes merging with Sakai tricky.